src/DataAccessLayer/InMemoryDal.cs

```diff
@@ -40,6 +42,15 @@ public InMemoryDal()
             LoadActors(settings);
             LoadGenres(settings);
             LoadMovies(settings);
+
+            // 16 bytes
+            benchmarkData = "0123456789ABCDEF";
+
+            // 1 MB
+            while (benchmarkData.Length < 1024 * 1024)
```

Comment:
we should look into changing this to Span<> or Memory<> as the mem copies will be fast with zero GC